scss/inc/_apip-text-to-speech.scss

```diff
@@ -64,30 +79,91 @@
     &.sfhMode {
         .tts-controls {
             .tts-control.tts-control-mode {
-                @include vendor-prefix(box-shadow, inset 2px 2px 4px rgba(0,0,0,0.20), property);
+                @include vendor-prefix(box-shadow, inset 2px 2px 4px rgba(0, 0, 0, 0.20), property);
 
                 background-color: darken($uiClickableDefaultBg, 10);
             }
         }
     }
+
     &.settings {
-      .tts-controls {
-          .tts-control-container {
-              .tts-slider-container {
-                  @include flex-container();
-                  @include vendor-prefix(align-items, center, property);
-              }
-
-              &:last-child {
-                  @include vendor-prefix(box-shadow, inset 2px 2px 4px rgba(0,0,0,0.20), property);
-
-                  background-color: darken($uiClickableDefaultBg, 10);
-
-                  &:hover {
-                      background-color: darken($uiClickableDefaultBg, 10);
-                  }
-              }
-          }
-      }
-  }
+        .tts-controls {
+            .tts-control-container {
+                .tts-slider-container {
+                    @include flex-container();
+                    @include vendor-prefix(align-items, center, property);
+                }
+
+                &:last-child {
+                    @include vendor-prefix(box-shadow, inset 2px 2px 4px rgba(0, 0, 0, 0.20), property);
+
+                    background-color: darken($uiClickableDefaultBg, 10);
+                }
+
+                .tts-control-settings {
+                    &:hover {
+                        background-color: transparent;
+                    }
+                }
+            }
+        }
+    }
+}
+
+.tts-content-node {
+    outline: none;
+}
+
+
+.tts-visible.tts-visible.tts-visible {
```

Comment:
What is the benefit of this construction? `.tts-visible` is repeated 3 times.
Shouldn't it be the same with only `.tts-visible { }` ?